lms/djangoapps/support/static/support/jsx/entitlements/components/Table/TableContainer.jsx

```diff
@@ -0,0 +1,14 @@
+import { connect } from 'react-redux';
+
+import EntitlementTable from './Table';
+
+const mapStateToProps = state => ({
+  entitlements: state.entitlements,
+});
+
+const TableContainer = connect(
+  mapStateToProps,
+  {},
+)(EntitlementTable);
+
+export default TableContainer;
```

Comment:
If it's `EntitlementTable` should this be `EntitlementTableContainer`?

Reply:
I had avoided naming the table Table simply because I didn't want to collide with the paragon 'Table'. Do you think it makes sense to be prefixing all my components with Entitlement_____, I hadn't done so for the Search component.

Reply:
When you import the Paragon `Table` you could use named imports to avoid name collision like

```javascript
import { Table as ParagonTable } from '@edx/paragon';

const Table = () => (<ParagonTable />);

export default Table;
```

Reply:
~~I think both `Table` and `EntitlementTable` are reasonable~~. Hmm, the more I think about it, maybe `EntitlementTable` makes more sense? 

Additionally, it feels like the table isn't just a table of `Entitlement`s - it also contains support actions that can be performed on these `Entitlement`s, so maybe there's even more description that can be added to this component.